pkg/cmd/propertiesGet_test.go

```diff
@@ -17,11 +17,12 @@ func TestPropertiesGetCommandInCommandCollectionHasName(t *testing.T) {
 	factory := NewMockFactory()
 	commands, _ := NewCommandCollection(factory)
 
-	propertiesGetCommand := commands.GetCommand(COMMAND_NAME_PROPERTIES_GET)
+	propertiesGetCommand, err := commands.GetCommand(COMMAND_NAME_PROPERTIES_GET)
 	assert.Equal(t, COMMAND_NAME_PROPERTIES_GET, propertiesGetCommand.Name())
 	assert.NotNil(t, propertiesGetCommand.Values())
 	assert.IsType(t, &PropertiesGetCmdValues{}, propertiesGetCommand.Values())
```

Comment:
Not sure if these asserts should be in an 'if err == nil' block ?

Reply:
error caught before everything, should exit out before hitting the other asserts now I think?